components/vf-sass-config/CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# 1.1.0
```

Comment:
I think the wrong changelog got updated.

Reply:
Oh damn! I'll make a PR to fix the bug

Reply:
Ah, have just whipped one up, hopefully you saw already: https://github.com/visual-framework/vf-core/pull/778/files